.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
While this is uncommon, some Windows systems are lacking `sh`. Is there a way to use Husky v7 for those?

Reply:
Yeah, I'm actually on Windows mostly and it works fine.

Reply:
Most Windows developers have `sh` installed, but not all.
Sometimes, an alternative is to use a Node script instead. However, I am wondering whether this is possible with Husky v7?

Reply:
This is unrelated to bash, I don't have bash at all. The changes you see here are from their official migration.

Reply:
Also, these are git hooks, which work fine everywhere.

Reply:
https://github.com/typicode/husky-4-to-7

Reply:
So, basically husky sets the `core.hooksPath` of the git repo to be `.husky`, hence how this works :)

Reply:
I agree that most Windows developers have `sh` or Bash installed through git for Windows or other similar solutions. However, I have encountered some Windows developers who do not. While this is uncommon, I am wondering whether there is a way to make it work for them. 

That being said, if Husky v7 does require `sh`, then this change sounds good to me.

What are your thoughts on this @erezrokah?

Reply:
@ehmicky please read my comments :)

Reply:
Not most, **everyone** who installs Git on Windows has MSYS2 -> bash. That's how Git works on Windows. This is nothing new, it's like that forever.

So, what you are seeing here is simply git hooks. How they are handled it's outside of the scope of the PR but I explained that above :)

Reply:
Thanks @XhmikosR, I forgot that installing `git` implied installing Bash as well. You're absolutely correct.

(Small note: let's make sure we use kind and welcoming language in such technical discussions, [in accordance to our code of conduct](https://github.com/netlify/eslint-config-node/blob/main/CODE_OF_CONDUCT.md#our-standards))